src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -590,6 +590,14 @@ internal void ProcessFunctionMetadataResponses(FunctionMetadataResponse function
                     });
                 }
             }
+            else
+            {
+                functions.Add(new RawFunctionMetadata()
+                {
+                    UseDefaultMetadataIndexing = functionMetadataResponse.UseDefaultMetadataIndexing
```

Comment:
renaming to `UseHostIndexing` would be little less ambiguous, Since default indexing can move between host and worker